README.md

```diff
@@ -78,6 +81,8 @@ jobs:
       run: /generate-primes.sh -d prime-numbers
 
     - name: Use Prime Numbers
+      env:
```

Comment:
This would need to be an `env` variable on the `cache-primes` step

Reply:
https://github.com/actions/cache/pull/489/commits/6ceddd55d128082f23eaf7026dfd38e5f14baa41